tmail-backend/integration-tests/imap/memory-imap-integration-tests/src/test/resources/pop3server.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<pop3servers>
```

Comment:
With the module chooser now, I think the tests would pass even without the `pop3server.xml` file?

Reply:
No
I think it first too
but `pop3server.xml` still exist somewhere, (from james, when we inherit contract test from james)
Then still need override `pop3server.xml` file here

Reply:
It is this file: https://github.com/apache/james-project/blob/master/server/protocols/protocols-pop3/src/main/resources/pop3server.xml#L22-L21

Seems like a redundant file to me.